app/src/main/java/org/apache/taverna/mobile/ui/DownloadingFragment.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.taverna.mobile.ui;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.apache.taverna.mobile.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+
+public class DownloadingFragment extends Fragment {
+
+    private static final String ARGS_MESSAGE = "args_message";
+
+    String message;
+
+    @BindView(R.id.tvMessage)
+    TextView tvMessage;
+
+    public static DownloadingFragment newInstance(String message) {
+
+        Bundle args = new Bundle();
+        args.putString(ARGS_MESSAGE, message);
+        DownloadingFragment fragment = new DownloadingFragment();
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        message = getArguments().getString(ARGS_MESSAGE);
```

Comment:
This can be raised null pointer exception, So Please check first getArguments() is not null.